system/ee/ExpressionEngine/View/_shared/sidebar/navigation/list_item.php

```diff
@@ -9,7 +9,7 @@
         </span>
     <?php endif; ?>
     <?php if ($addlink) : ?>
-    <a href="<?=$addlink?>" class="dropdown__item-button button button--secondary button--xsmall"<?=$addlinkAttibutes?>><i class="fal fa-plus"></i><span class="hidden"><?=lang('btn_create_new') . ' ' . $text?></span></a>
+    <a href="<?=$addlink?>" class="dropdown__item-button button button--secondary button--xsmall"<?=$addlinkAttributes?>><i class="fal fa-plus"></i><span class="hidden"><?=lang('btn_create_new') . ' ' . $text?></span></a>
```

Comment:
Not sure we can change this one? @matthewjohns0n

Reply:
I did update this everywhere, but since it is a shared form view, we probably cant make this change.